core/src/main/scala/io/finch/endpoint/effect/Bodies.scala

```diff
@@ -0,0 +1,81 @@
+package io.finch.endpoint.effect
+
+import com.twitter.concurrent.AsyncStream
+import com.twitter.io.Buf
+import io.finch._
+import scala.reflect.ClassTag
+
+trait Bodies[F[_]] { self: EffectInstances[F] =>
```

Comment:
I had to copypaste all the methods once again into `effect` subpackage because from a customer perspective writing something like `body[Rerunnable, A]` for every endpoint is too much, so when `io.finch.%effectName%._` is imported, only all those copy-pasted endpoints are available with `F` already defined by `%effectName%`